app/src/main/java/com/dessalines/thumbkey/ui/components/settings/SettingsActivity.kt

```diff
@@ -95,7 +94,7 @@ fun SettingsActivity(
     val scrollState = rememberScrollState()
     var showConfirmResetDialog by remember { mutableStateOf(false) }
 
-    val layoutsState = remember { mutableStateOf(keyboardLayoutsSetFromDbIndexString(settings?.keyboardLayouts)) }
```

Comment:
Removing these remember was key, and was a leftover from the old settings library.